letsencrypt/client/tests/revoker_test.py

```diff
@@ -0,0 +1,390 @@
+"""Test :mod:`letsencrypt.client.revoker`."""
```

Comment:
Nitpick: That's a precedence. Can we leave everything consistent for now,  i.e. `"""Tests for letsencrypt.client.revoker."""` and update in all places at once (if necessary)?